apps/next-evans/pages/getting-started.tsx

```diff
@@ -0,0 +1,351 @@
+import recommendedBooks from '@/lib/recommended-books';
+import { mostModernEdition } from '@/lib/editions';
+import getCustomCode from '@/lib/get-custom-code';
+
+export const getStaticProps: GetStaticProps<Props> = async () => {
+  const prisma = new PrismaClient();
+
+  const friends = await prisma.friends.findMany({
+    where: { lang: LANG },
+    select: {
+      slug: true,
+      name: true,
+      documents: {
+        select: {
+          slug: true,
+          title: true,
+          editions: {
+            select: {
+              type: true,
+              edition_audios: {
+                select: {
+                  id: true,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  });
+
+  const customCode = await Promise.all(
+    friends.map((friend) =>
+      getCustomCode(
+        friend.slug,
+        friend.documents.map((doc) => doc.slug),
+      ),
+    ),
+  );
+
+  const allBooks: NeededCoverProps[] = friends.flatMap((friend, index) => {
+    const codeForFriend = customCode[index];
+    return friend.documents.map((doc) => {
+      const codeForDocument = codeForFriend ? codeForFriend[doc.slug] : undefined;
+      return {
+        authorSlug: friend.slug,
+        author: friend.name,
+        title: doc.title,
+        edition: mostModernEdition(doc.editions.map((edition) => edition.type)),
+        customCss: codeForDocument ? codeForDocument.css || `` : ``,
+        customHtml: codeForDocument ? codeForDocument.html || `` : ``,
+        hasAudio: doc.editions.some((edition) => edition.edition_audios),
+        slug: doc.slug,
+      };
+    });
+  });
+
+  function filterBooks(
+    books: Array<NeededCoverProps>,
+    category: 'history' | 'doctrine' | 'spiritualLife' | 'journals',
+  ): Array<NeededCoverProps> {
+    return books
+      .filter((book) =>
+        recommendedBooks[category][LANG].some(
+          (recommendedBook) =>
+            recommendedBook.author === book.authorSlug &&
+            recommendedBook.title === book.slug,
+        ),
+      )
+      .sort(
+        (a, b) =>
+          recommendedBooks[category][LANG].findIndex(
+            (book) => book.title === a.slug && book.author === a.authorSlug,
+          ) -
+          recommendedBooks[category][LANG].findIndex(
+            (book) => book.title === b.slug && book.author === b.authorSlug,
+          ),
+      );
+  }
+
+  const organizedBooks = {
+    history: filterBooks(allBooks, `history`),
+    doctrine: filterBooks(allBooks, `doctrine`),
+    spiritualLife: filterBooks(allBooks, `spiritualLife`),
+    journals: filterBooks(allBooks, `journals`),
+  };
+
+  return { props: { books: organizedBooks, numBooks: allBooks.length } };
+};
+
+export type NeededCoverProps = Pick<
+  CoverProps,
+  'title' | 'author' | 'edition' | 'customCss' | 'customHtml'
+> & {
+  authorSlug: string;
+  hasAudio: boolean;
+  slug: string;
```

Comment:
is this the document slug? could you modify it's name so it's clear what sort of slug it is?